app/services/decisions/outgoings_decision_tree.rb

```diff
@@ -22,15 +22,11 @@ def destination # rubocop:disable Metrics/MethodLength
     private
 
     def after_housing_payment_type
-      if form_object.housing_payment_type.nil?
-        # TODO: Consider appropriate action for empty housing_payment_type
+      case form_object.housing_payment_type&.value
+      when :board_and_lodging, :mortgage, :rent
+        edit(form_object.housing_payment_type.value)
+      when nil, :none
         edit(:council_tax)
```

Comment:
Should it go to council tax next if there's no housing payment type? Wouldn't it just skip to the page after? Or is this assuming that if they don't select any of the housing payment type options then they must own their home?